codalab/lib/worksheet_util.py

```diff
@@ -379,100 +375,6 @@ def friendly_render_dep(dep):
     return None
 
 
-def interpret_file_genpaths(client, requests):
```

Comment:
Do we have a way to think about which stuff goes in `worksheet_util.py` and what goes in `interpet,py`?  Maybe would be good to document this.